install.sh

```diff
@@ -103,24 +103,29 @@ check_download() {
 
     # Checkout the correct stream
     cd $1
-    git checkout origin/${STREAM}
+    git checkout ${STREAM}
 }
 
 check_download wayfire
 check_download wf-shell
 
 cd $BUILDROOT/wayfire
-
+[ -d build ] && rm -rf build
```

Comment:
Why this? Didn't you want to have possibility for kind of incremental build, if some stage of the script fails? If the directories are to be deleted, then the user is supposed to use --clean.

Reply:
It still reruns meson and ninja when rerunning the script. If it has failed previously, the next time it is run, meson will error that the directory is already configured. I believe there is some caching even when the build directory is deleted, so subsequent runs are quicker.

Reply:
The caching comes from ccache, if anything. Maybe we should always do a clean build then, and drop the option.